Content.Shared/Electrocution/Components/ElectrifiedComponent.cs

```diff
@@ -1,121 +1,125 @@
+using Robust.Shared.GameStates;
 using Robust.Shared.Audio;
 
-namespace Content.Server.Electrocution;
+namespace Content.Shared.Electrocution;
 
 /// <summary>
 ///     Component for things that shock users on touch.
 /// </summary>
-[RegisterComponent]
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
 public sealed partial class ElectrifiedComponent : Component
```

Comment:
Moved to shared, marked as networked, no system code was moved to shared tho. It looks ok but i was strongly discouraged from doing this... I'd be very happy if i had idea on 'how' - moving system and making predictable ElectrocutionSystem is by all means no easy task